src/OrbitQt/CaptureOptionsDialog.ui

```diff
@@ -541,6 +541,48 @@ p, li { white-space: pre-wrap; }
             </property>
            </widget>
           </item>
+          <item>
+           <widget class="QCheckBox" name="threadStateChangeCallstackCollectionCheckBox">
+            <property name="text">
+             <string>Collect call stacks on thread state changes</string>
+            </property>
+            <property name="checked">
+             <bool>true</bool>
+            </property>
+           </widget>
+          </item>
+          <item>
+           <widget class="QGroupBox" name="threadStateChangeCallstackUnwindingMethodGroupBox">
+            <property name="enabled">
+             <bool>true</bool>
+            </property>
+            <property name="title">
+             <string>Unwinding method</string>
```

Comment:
We've already had a Unwinding method group box, I found it a bit strange to have another one and that the options are not related at all. Consider explaining this by a tooltip as I mentioned before and also changing its name.

Reply:
Added.